doc-content/enterprise-only/project-data/preview-editor-download-test-proc.adoc

```diff
@@ -0,0 +1,9 @@
+[id='preview-editor-download-test-proc']
+= Downloading a test scenario
+
+You can also download a copy of the test scenario to your local machine for future reference or as backup.
+
+.Procedure
+. From the *Test Scenario (Preview)* editor toolbar on the upper-right, click *Download*.
+
+The .scesim file will be downloaded to your local machine.
```

Comment:
Avoid "will be".  Suggestion: The .scesim file is downloaded to your local machine.
Make global correction in the doc. Avoid "will be".